bfm/CMakeLists.txt

```diff
@@ -15,20 +15,31 @@
 # You should have received a copy of the GNU Lesser General Public
 # License along with this library; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+#
 
 cmake_minimum_required(VERSION 3.6)
 project(bfm C CXX)
+include(${BF_FLAGS_DIR}/flags.cmake)
+
+set(CMAKE_ASM_NASM_OBJECT_FORMAT "elf64")
+enable_language(ASM_NASM)
```

Comment:
This doesn't seem like it would work on ARM. Also, why is this not in it's own `nasm.cmake`?